src/main/java/com/marianbastiurea/lifeofbees/Hive.java

```diff
@@ -165,6 +158,33 @@ public List<EggsFrame> getEggsFrames() {
         return eggsFrames;
     }
 
+    public double ageOfQueenIndex(Hive hive) {
```

Comment:
This method is a mess. You can remove it anyway, it does nothing.

Reply:
done